packages/cms-lib/errorHandlers.js

```diff
@@ -300,6 +304,45 @@ function logFileSystemErrorInstance(error, context) {
   debugErrorAndContext(error, context);
 }
 
+/**
+ * Logs a message for an error instance resulting from API interaction
+ * related to serverless function.
+ *
+ * @param {Error|SystemError|Object} error
+ * @param {ApiErrorContext}          context
+ */
+function logServerlessFunctionApiErrorInstance(error, scopesData, context) {
```

Comment:
I will fix the scopesData issue.

Initially I wanted to make this async and fetch the data within, but set it up the way that I did to avoid a circular dependency issue that importing `http` creates.